examples/surveyor/ushou-surveyor-etc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "uuid": "a9196c6b-ac5a-4307-b45f-d93e78e84351",
+  "ship": {
+    "imo": "9634098"
+  },
+  "port": "USHOU",
+  "eventType": "surveyor.etc.serviceProvider",
+  "eventTime": "2020-01-01T00:00:00Z",
```

Comment:
To leave no room for confusion. Perhaps use an event time that is not identical to the example ETS event time?